api/policy.go

```diff
@@ -193,6 +202,28 @@ func UnmarshalSQLReviewPolicy(payload string) (*advisor.SQLReviewPolicy, error)
 	return &sr, nil
 }
 
+// EnvironmentTierPolicy is the tier of an environment.
+type EnvironmentTierPolicy struct {
+	EnvironmentTierList []EnvironmentTierValue `json:"environmentTierList"`
```

Comment:
Any reason for this to be a list? I guess this intention is to handle the empty value? We can use a default value UNPROTECTED instead.